lib/mixlib/install/options.rb

```diff
@@ -25,17 +25,7 @@ class Install
     class Options
       class InvalidOptions < ArgumentError; end
 
-      attr_reader :options, :errors, :original_platform_version
-
-      SUPPORTED_ARCHITECTURES = %w{
-        i386
-        powerpc
-        ppc64
-        ppc64le
-        s390x
-        sparc
-        x86_64
-      }
```

Comment:
This will be a fixed list and thus we should do validation on that fixed liset for an improved UX in mixlib-install and the tools that use it like chef-ingredient.